packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -1764,6 +1764,228 @@ RED.nodes = (function() {
+            newNode.wires = node.wires || [];
+            newNode.inputLabels = node.inputLabels;
+            newNode.outputLabels = node.outputLabels;
+            newNode.icon = node.icon;
+        }
+        if (node.type === "junction") {
+            newNode.wires = node.wires || [];
+        }
+        if (node.hasOwnProperty("l")) {
+            newNode.l = node.l;
+        }
+        if (node.hasOwnProperty("d")) {
+            newNode.d = node.d;
+        }
+        if (node.hasOwnProperty("g")) { // Group
+            newNode.g = node.g;
+        }
+        if (options.markChanged) {
+            newNode.changed = true
+        }
+
+        if (node.type === "group") {
+            newNode._def = RED.group.def;
+            for (const d in newNode._def.defaults) {
+                if (newNode._def.defaults.hasOwnProperty(d) && d !== "inputs" && d !== "outputs") {
+                    newNode[d] = node[d];
+                    newNode._config[d] = JSON.stringify(node[d]);
+                }
+            }
+            newNode._config.x = node.x;
+            newNode._config.y = node.y;
+            if (node.hasOwnProperty("w")) { // Weight
+                newNode.w = node.w
+            }
+            if (node.hasOwnProperty("h")) { // Height
+                newNode.h = node.h
+            }
+        } else if (node.type === "junction") {
+            newNode._def = { defaults: {} };
+            newNode._config.x = node.x
+            newNode._config.y = node.y
+            newNode.inputs = 1
+            newNode.outputs = 1
+            newNode.w = 0;
+            newNode.h = 0;
+        } else if (node.type.substring(0, 7) === "subflow") {
+            const parentId = node.type.split(":")[1];
+            /*
+            const subflow = subflow_denylist[parentId]||subflow_map[parentId]||getSubflow(parentId);
+            // TODO: Pourquoi ne pas l'avoir mis dans les tabs ?
+            if (!subflow){
+                node._def = {
+                    color:"#fee",
+                    defaults: {},
+                    label: "unknown: "+n.type,
+                    labelStyle: "red-ui-flow-node-label-italic",
+                    outputs: n.outputs|| (n.wires && n.wires.length) || 0,
+                    set: registry.getNodeSet("node-red/unknown")
+                }
+            } else {
+                // TODO: Normal que la copie utilise le Subflow parent ?
+                if (subflow_denylist[parentId] || createNewIds || options.importMap[n.id] === "copy") {
+                    parentId = subflow.id;
+                    node.type = "subflow:"+parentId;
+                    node._def = registry.getNodeType(node.type);
+                    delete node.i;
+                }
+                node.name = n.name;
+                node.outputs = subflow.out.length;
+                node.inputs = subflow.in.length;
+                node.env = n.env;
+            }*/
```

Comment:
Adding the missing subflow can be done before (L2084 or L2124) centrally.

I guess it's normal for the parent subflow to be used when copying?